dev/tracing_tests/README.md

```diff
@@ -4,4 +4,4 @@ The tests in this folder must be run with `flutter test --enable-vmservice`,
 since they test that trace data is written to the timeline by connecting to
 the observatory.
 
-These tests will fail if run without this flag.
\ No newline at end of file
+These tests will fail if run without this flag.
```

Comment:
I can't spot what has changed on this line...

Reply:
new line at EOF

Reply:
Ah, I thought github's UI actually showed something to visualize that... Maybe I dreamt that.

Reply:
It puts a little icon at the end if it's missing, but I guess not when you add it

Reply:
I only noticed because I `cat`ed the file and it didn't print nicely